### DISCUSSION TITLE
How to download package

### DISCUSSION THREAD

**Discussion by architvarma** [community/community#66142] [Packages]
Hi I'm trying to download maven package from GitHub Package with the help of curl, wget but neither of the command is working can anyone suggest how to download maven package with the help of curl wget or any other command ?

**Comment by jb-2020** [comment]
+1 It would be fantastic if the runner came with just a few of the most basic utilities IMO.  `curl` being number 1 on my list followed by GNU gettext

Edit: Mistakenly thought this was related to the lack of utilities in the Github Actions runner image.
